tests/integration/components/donation/credit-card-test.js

```diff
@@ -19,22 +19,21 @@ let setHandler = function(context, handlerName, handler = K) {
   context.set(handlerName, handler);
 };
 
-let getFreshDates = function() {
-  let date = new Date();
-  let currentMonth = `0${date.getMonth() + 1}`.slice(-2);
-  let currentYear = date.getFullYear();
-
-  return {
-    month: currentMonth,
-    year: currentYear
-  };
-};
+// let getFreshDates = function() {
+//   let date = new Date();
+//   let currentMonth = `0${date.getMonth() + 1}`.slice(-2);
+//   let currentYear = date.getFullYear();
+//
+//   return {
+//     month: currentMonth,
+//     year: currentYear
+//   };
+// };
```

Comment:
Looks like this is unused now that "cancel" is gone. Feel free to remove the code as well.